_ext/listenbrainz.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Mopidy-Listenbrainz
+type: frontend
+dev:
+  github: suaviloquence/mopidy-listenbrainz
+dist:
+  pypi: Mopidy-Listenbrainz
+logo: /media/ext/listenbrainz.svg
+---
+
+Extension for recording song listens to [Listenbrainz](https://listenbrainz.org), a libre alternative to
+Last.fm.  It also provides Listenbrainz-generated recommendation playlists to Mopidy.
```

Comment:
You're free to capitalize your project name as you see fit, but it looks like "Listenbrainz" should have a capitalized B:

```suggestion
Extension for recording song listens to [ListenBrainz](https://listenbrainz.org), a libre alternative to
Last.fm.  It also provides ListenBrainz-generated recommendation playlists to Mopidy.
```